### DISCUSSION TITLE
What is your top 3 favorite theme?

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#528] [General]
What is your top 3 favorite theme?
___

Here is my top 3:

1. Sunken
2. Cucumber Splash
3. Sky

**Comment by Taiko2k** [comment]
I'm glad you like sunken, it was one of the hardest to make.

Here's mine:

 1. Astro
 2. Turbo
 3. Rose

Though I also use Carbon and Vape sometimes.

**Reply by Solarunit** [reply]
zero matches  :smile:
